lib/signaling/v2/peerconnection.js

```diff
@@ -91,6 +92,12 @@ function PeerConnectionV2(id, encodingParameters, preferredCodecs, options) {
   var peerConnection = new RTCPeerConnection(configuration);
 
   Object.defineProperties(this, {
+    _dataChannels: {
```

Comment:
Can we keep this as `_dataTrackSenders`?

Reply:
@markandrus We actually store `RTCDataChannel`s in this collection. So isn't this a more appropriate name?

Reply:
OK, good point 👍